main.go

```diff
@@ -61,6 +61,8 @@ func main() {
 		controllers.DefaultResticContainerImage, "The container image for the restic data mover")
 	flag.StringVar(&controllers.RsyncContainerImage, "rsync-container-image",
 		controllers.DefaultRsyncContainerImage, "The container image for the rsync data mover")
+	flag.StringVar(&controllers.ScribeName, "scribe-name",
+		controllers.DefaultScribeName, "The name with which rsync data mover resources will be prefixed")
```

Comment:
Is this only specific to *rsync* ?

Reply:
Rclone needs this `scc` too

Reply:
changed to `The name with which scribe resources will be prefixed`

Reply:
updated to specify SCC name, only